.github/release-drafter.yml

```diff
@@ -0,0 +1,75 @@
+name-template: 'v$RESOLVED_VERSION'
+tag-template: 'v$RESOLVED_VERSION'
+categories:
+  - title: '🚀 Features'
+    labels:
+      - 'feature'
+  - title: '🛸 Enhancements'
+    labels:
+      - 'enhancement'
+  - title: '🐛 Bug Fixes'
+    labels:
+      - 'fix'
+      - 'bugfix'
+  - title: '🧰 Maintenance'
+    label: 'chore'
+  - title: '📦 Dependency Updates'
+    label: 'dependencies'
+  - title: '📖 Documentation'
+    label: 'docs'
+version-resolver:
+  major:
+    labels:
+      - 'major'
+  minor:
+    labels:
+      - 'minor'
+  patch:
+    labels:
+      - 'patch'
+  default: patch
+autolabeler:
+  - label: 'enhancement'
+    branch:
+      - '/refactor\/.+/'
+    title:
+      - '/refactor/i'
```

Comment:
why is refactor seen as an enhancement?

Reply:
I suppose if you're performing a refactor that it's inherently about modifying existing code, and so if it's not for a bug fix that it must be some sort of enhancement, otherwise why else would you change it 🤷🏽